### ISSUE TITLE
Vagrant Up not working with supplied VagrantFile

### ISSUE THREAD

**Issue by kershnerd** [zendframework/ZendSkeletonApplication#385]
I am on a MAC with the latest version of VirtualBox, vagrant and composer.  This is what the output is when I Vagrant Up:

Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'ubuntu/xenial64'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'ubuntu/xenial64' is up to date...
==> default: Setting the name of the VM: guitarsessions_default_1473801907008_59089
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 80 (guest) => 8080 (host) (adapter 1)
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: ubuntu
    default: SSH auth method: password
    default: 
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
==> default: Mounting shared folders...
    default: /var/www => /Users/kershnerd/Projects/guitarsessions
==> default: Running provisioner: shell...
    default: Running: inline script
==> default: mesg: 
==> default: ttyname failed
==> default: : 
==> default: Inappropriate ioctl for device
==> default: Cannot add PPA: 'ppa:~ondrej/ubuntu/php'.
==> default: ERROR: '~ondrej' user or team does not exist.
==> default: Err:1 http://archive.ubuntu.com/ubuntu xenial InRelease
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:2 http://archive.ubuntu.com/ubuntu xenial-updates InRelease
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:3 http://archive.ubuntu.com/ubuntu xenial-backports InRelease
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:4 http://security.ubuntu.com/ubuntu xenial-security InRelease
==> default:   Temporary failure resolving 'security.ubuntu.com'
==> default: Reading package lists...
==> default: W
==> default: : 
==> default: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/xenial/InRelease  Temporary failure resolving 'archive.ubuntu.com'
==> default: W
==> default: : 
==> default: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/xenial-updates/InRelease  Temporary failure resolving 'archive.ubuntu.com'
==> default: W
==> default: : 
==> default: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/xenial-backports/InRelease  Temporary failure resolving 'archive.ubuntu.com'
==> default: W
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/dists/xenial-security/InRelease  Temporary failure resolving 'security.ubuntu.com'
==> default: W
==> default: : 
==> default: Some index files failed to download. They have been ignored, or old ones used instead.
==> default: Reading package lists...
==> default: Building dependency tree...
==> default: Reading state information...
==> default: git is already the newest version (1:2.7.4-0ubuntu1).
==> default: curl is already the newest version (7.47.0-1ubuntu2.1).
==> default: The following additional packages will be installed:
==> default:   apache2-bin apache2-data apache2-utils libapr1 libaprutil1
==> default:   libaprutil1-dbd-sqlite3 libaprutil1-ldap libcurl3 liblua5.1-0 libxslt1.1
==> default:   libzip4 php-common php7.0-common php7.0-readline ssl-cert
==> default: Suggested packages:
==> default:   www-browser apache2-doc apache2-suexec-pristine | apache2-suexec-custom
==> default:   php-pear openssl-blacklist
==> default: The following NEW packages will be installed:
==> default:   apache2 apache2-bin apache2-data apache2-utils libapache2-mod-php7.0 libapr1
==> default:   libaprutil1 libaprutil1-dbd-sqlite3 libaprutil1-ldap libcurl3 liblua5.1-0
==> default:   libxslt1.1 libzip4 php-common php7.0 php7.0-bcmath php7.0-bz2 php7.0-cli
==> default:   php7.0-common php7.0-curl php7.0-intl php7.0-json php7.0-mbstring
==> default:   php7.0-opcache php7.0-readline php7.0-soap php7.0-sqlite3 php7.0-xml
==> default:   php7.0-xsl php7.0-zip ssl-cert
==> default: 0 upgraded, 31 newly installed, 0 to remove and 0 not upgraded.
==> default: Need to get 6,274 kB of archives.
==> default: After this operation, 24.7 MB of additional disk space will be used.
==> default: Err:1 http://archive.ubuntu.com/ubuntu xenial/main amd64 libapr1 amd64 1.5.2-3
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:2 http://archive.ubuntu.com/ubuntu xenial/main amd64 libaprutil1 amd64 1.5.4-1build1
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:3 http://archive.ubuntu.com/ubuntu xenial/main amd64 libaprutil1-dbd-sqlite3 amd64 1.5.4-1build1
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:4 http://archive.ubuntu.com/ubuntu xenial/main amd64 libaprutil1-ldap amd64 1.5.4-1build1
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:5 http://archive.ubuntu.com/ubuntu xenial/main amd64 liblua5.1-0 amd64 5.1.5-8ubuntu1
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:6 http://security.ubuntu.com/ubuntu xenial-security/main amd64 apache2-bin amd64 2.4.18-2ubuntu3.1
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Ign:7 http://security.ubuntu.com/ubuntu xenial-security/main amd64 apache2-utils amd64 2.4.18-2ubuntu3.1
==> default: Ign:8 http://security.ubuntu.com/ubuntu xenial-security/main amd64 apache2-data all 2.4.18-2ubuntu3.1
==> default: Ign:9 http://security.ubuntu.com/ubuntu xenial-security/main amd64 apache2 amd64 2.4.18-2ubuntu3.1
==> default: Err:10 http://archive.ubuntu.com/ubuntu xenial/main amd64 php-common all 1:35ubuntu6
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Ign:11 http://security.ubuntu.com/ubuntu xenial-security/main amd64 php7.0-common amd64 7.0.8-0ubuntu0.16.04.2
==> default: Ign:12 http://security.ubuntu.com/ubuntu xenial-security/main amd64 php7.0-json amd64 7.0.8-0ubuntu0.16.04.2
==> default: Ign:13 http://security.ubuntu.com/ubuntu xenial-security/main amd64 php7.0-opcache amd64 7.0.8-0ubuntu0.16.04.2
==> default: Ign:14 http://security.ubuntu.com/ubuntu xenial-security/main amd64 php7.0-readline amd64 7.0.8-0ubuntu0.16.04.2
==> default: Ign:15 http://security.ubuntu.com/ubuntu xenial-security/main amd64 php7.0-cli amd64 7.0.8-0ubuntu0.16.04.2
==> default: Err:6 http://security.ubuntu.com/ubuntu xenial-security/main amd64 apache2-bin amd64 2.4.18-2ubuntu3.1
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:7 http://security.ubuntu.com/ubuntu xenial-security/main amd64 apache2-utils amd64 2.4.18-2ubuntu3.1
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:8 http://security.ubuntu.com/ubuntu xenial-security/main amd64 apache2-data all 2.4.18-2ubuntu3.1
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:9 http://security.ubuntu.com/ubuntu xenial-security/main amd64 apache2 amd64 2.4.18-2ubuntu3.1
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:11 http://security.ubuntu.com/ubuntu xenial-security/main amd64 php7.0-common amd64 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:12 http://security.ubuntu.com/ubuntu xenial-security/main amd64 php7.0-json amd64 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:13 http://security.ubuntu.com/ubuntu xenial-security/main amd64 php7.0-opcache amd64 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:14 http://security.ubuntu.com/ubuntu xenial-security/main amd64 php7.0-readline amd64 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:15 http://security.ubuntu.com/ubuntu xenial-security/main amd64 php7.0-cli amd64 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:16 http://security.ubuntu.com/ubuntu xenial-security/main amd64 libapache2-mod-php7.0 amd64 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Ign:17 http://security.ubuntu.com/ubuntu xenial-security/main amd64 libcurl3 amd64 7.47.0-1ubuntu2.1
==> default: Err:18 http://archive.ubuntu.com/ubuntu xenial/main amd64 libxslt1.1 amd64 1.1.28-2.1
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:19 http://archive.ubuntu.com/ubuntu xenial/universe amd64 libzip4 amd64 1.0.1-0ubuntu1
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Ign:20 http://security.ubuntu.com/ubuntu xenial-security/main amd64 php7.0 all 7.0.8-0ubuntu0.16.04.2
==> default: Ign:21 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 php7.0-bcmath amd64 7.0.8-0ubuntu0.16.04.2
==> default: Ign:22 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 php7.0-bz2 amd64 7.0.8-0ubuntu0.16.04.2
==> default: Ign:23 http://security.ubuntu.com/ubuntu xenial-security/main amd64 php7.0-curl amd64 7.0.8-0ubuntu0.16.04.2
==> default: Ign:24 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 php7.0-intl amd64 7.0.8-0ubuntu0.16.04.2
==> default: Ign:25 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 php7.0-mbstring amd64 7.0.8-0ubuntu0.16.04.2
==> default: Err:16 http://security.ubuntu.com/ubuntu xenial-security/main amd64 libapache2-mod-php7.0 amd64 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:17 http://security.ubuntu.com/ubuntu xenial-security/main amd64 libcurl3 amd64 7.47.0-1ubuntu2.1
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:20 http://security.ubuntu.com/ubuntu xenial-security/main amd64 php7.0 all 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:21 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 php7.0-bcmath amd64 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:22 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 php7.0-bz2 amd64 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:23 http://security.ubuntu.com/ubuntu xenial-security/main amd64 php7.0-curl amd64 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:24 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 php7.0-intl amd64 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:25 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 php7.0-mbstring amd64 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:26 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 php7.0-soap amd64 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Ign:27 http://security.ubuntu.com/ubuntu xenial-security/main amd64 php7.0-sqlite3 amd64 7.0.8-0ubuntu0.16.04.2
==> default: Ign:28 http://security.ubuntu.com/ubuntu xenial-security/main amd64 php7.0-xml amd64 7.0.8-0ubuntu0.16.04.2
==> default: Ign:29 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 php7.0-xsl all 7.0.8-0ubuntu0.16.04.2
==> default: Ign:30 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 php7.0-zip amd64 7.0.8-0ubuntu0.16.04.2
==> default: Err:31 http://archive.ubuntu.com/ubuntu xenial/main amd64 ssl-cert all 1.0.37
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:26 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 php7.0-soap amd64 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:27 http://security.ubuntu.com/ubuntu xenial-security/main amd64 php7.0-sqlite3 amd64 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:28 http://security.ubuntu.com/ubuntu xenial-security/main amd64 php7.0-xml amd64 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:29 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 php7.0-xsl all 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: Err:30 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 php7.0-zip amd64 7.0.8-0ubuntu0.16.04.2
==> default:   Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/a/apr/libapr1_1.5.2-3_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/a/apr-util/libaprutil1_1.5.4-1build1_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/a/apr-util/libaprutil1-dbd-sqlite3_1.5.4-1build1_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/a/apr-util/libaprutil1-ldap_1.5.4-1build1_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/l/lua5.1/liblua5.1-0_5.1.5-8ubuntu1_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/a/apache2/apache2-bin_2.4.18-2ubuntu3.1_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/a/apache2/apache2-utils_2.4.18-2ubuntu3.1_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/a/apache2/apache2-data_2.4.18-2ubuntu3.1_all.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/a/apache2/apache2_2.4.18-2ubuntu3.1_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/php-defaults/php-common_35ubuntu6_all.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/php7.0/php7.0-common_7.0.8-0ubuntu0.16.04.2_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/php7.0/php7.0-json_7.0.8-0ubuntu0.16.04.2_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/php7.0/php7.0-opcache_7.0.8-0ubuntu0.16.04.2_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/php7.0/php7.0-readline_7.0.8-0ubuntu0.16.04.2_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/php7.0/php7.0-cli_7.0.8-0ubuntu0.16.04.2_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/php7.0/libapache2-mod-php7.0_7.0.8-0ubuntu0.16.04.2_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/c/curl/libcurl3_7.47.0-1ubuntu2.1_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/libx/libxslt/libxslt1.1_1.1.28-2.1_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/universe/libz/libzip/libzip4_1.0.1-0ubuntu1_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/php7.0/php7.0_7.0.8-0ubuntu0.16.04.2_all.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/universe/p/php7.0/php7.0-bcmath_7.0.8-0ubuntu0.16.04.2_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/universe/p/php7.0/php7.0-bz2_7.0.8-0ubuntu0.16.04.2_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/php7.0/php7.0-curl_7.0.8-0ubuntu0.16.04.2_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/universe/p/php7.0/php7.0-intl_7.0.8-0ubuntu0.16.04.2_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/universe/p/php7.0/php7.0-mbstring_7.0.8-0ubuntu0.16.04.2_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/universe/p/php7.0/php7.0-soap_7.0.8-0ubuntu0.16.04.2_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/php7.0/php7.0-sqlite3_7.0.8-0ubuntu0.16.04.2_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/php7.0/php7.0-xml_7.0.8-0ubuntu0.16.04.2_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/universe/p/php7.0/php7.0-xsl_7.0.8-0ubuntu0.16.04.2_all.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/universe/p/php7.0/php7.0-zip_7.0.8-0ubuntu0.16.04.2_amd64.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/s/ssl-cert/ssl-cert_1.0.37_all.deb  Temporary failure resolving 'archive.ubuntu.com'
==> default: E
==> default: : 
==> default: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
==> default: /tmp/vagrant-shell: line 26: /etc/apache2/sites-available/000-default.conf: No such file or directory
==> default: /tmp/vagrant-shell: line 27: a2enmod: command not found
==> default: Failed to restart apache2.service: Unit apache2.service not found.
==> default: /tmp/vagrant-shell: line 33: php: command not found
==> default: curl: (6) Could not resolve host: getcomposer.org
==> default: ** [ZF] Run the following command to install dependencies, if you have not already:
==> default:     vagrant ssh -c 'composer install'
==> default: ** [ZF] Visit http://localhost:8080 in your browser for to view the application **

**Comment by webimpress** [comment]
@kershnerd it looks like temporary issue with DNS or your network:

> Temporary failure resolving 'archive.ubuntu.com'

Please try again, destroy it first and try one more time.

**Comment by kershnerd** [comment]
I have done this a ton of times (destroying and recreating).  From my MAC, it resolves when I do this:

curl archive.ubuntu.com

But when I vagrant ssh and run that curl from inside the vagrant box, I get this:
curl: (6) Could not resolve host: archive.ubuntu.com

I assume the box is not set up correctly for networking?  All curl requests fail for all websites I tried.

This is from inside the box:
ubuntu@ubuntu-xenial:/var/www$ ifconfig
enp0s3    Link encap:Ethernet  HWaddr 02:76:61:22:8c:2b  
          inet addr:10.0.2.15  Bcast:10.0.2.255  Mask:255.255.255.0
          inet6 addr: fe80::76:61ff:fe22:8c2b/64 Scope:Link
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:849 errors:0 dropped:0 overruns:0 frame:0
          TX packets:620 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000 
          RX bytes:85071 (85.0 KB)  TX bytes:107982 (107.9 KB)

lo        Link encap:Local Loopback  
          inet addr:127.0.0.1  Mask:255.0.0.0
          inet6 addr: ::1/128 Scope:Host
          UP LOOPBACK RUNNING  MTU:65536  Metric:1
          RX packets:2412 errors:0 dropped:0 overruns:0 frame:0
          TX packets:2412 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1 
          RX bytes:193736 (193.7 KB)  TX bytes:193736 (193.7 KB)

Any suggestions?

**Comment by kershnerd** [comment]
I think the problem is with this:
Inappropriate ioctl for device

The network device is enp0s3 no eth0

I have no idea how to fix this.

**Comment by webimpress** [comment]
@kershnerd I just tested on VirtualBox 5.0.26 and all was good.
Then I've updated VirtualBox to 5.1.6 and I have the same issue as you.
I would suggest downgrade VirtualBox to 5.0.26.

**Comment by kershnerd** [comment]
I downgraded to Virtual Box 5.0.26 and have the same problem.

I am using vagrant version 1.8.5 and composer version 1.2.1.  Does that match your versions?

**Comment by kershnerd** [comment]
Any other suggestions?  I don't know enough about Vagrant, but this VagrantFile doesn't allow Ubuntu to connect to the Internet.  Can I import a different box?

**Comment by kershnerd** [comment]
@webimpress I bet the reason it worked for you was because you have an order version of the ubuntu/xenial64 on your VirtualBox.  This box was updated 5 days ago and then again 2 hours ago.  Maybe something changed...

**Comment by webimpress** [comment]
@kershnerd  Yes, I have the same versions. I was testing on 5.0.26 `ubuntu/xenial64` box in version `20160805.0.0`, then I've updated everything and version `20160909.0.0` doesn't work. I think the problem is with the box. I've downgraded to version `20160805.0.0` again and now it is not working. They released also new version today `20160914.0.0`, but it is not working for me...

I've found this: https://github.com/mitchellh/vagrant/issues/7155#issuecomment-228568200
I've tried then `bento/ubuntu-16.04` v2.2.9 and seems to be fine (needs some changes - user is `vagrant` not `ubuntu`) - definitely network is fine, even on VirtualBox 5.1.6.

Try this and let me know. Maybe we should use `bento` images in this repository instead of official ubuntu....
